modules/nw-egress-router-redirect-mode-ovn.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc
+
+[id="nw-egress-router-redirect-mode-ovn_{context}"]
+= Deploying an egress router pod in redirect mode
```

Comment:
Curious why you don't use the {router-type} attribute in this module. I confess it's over my simple caveman head and perhaps has something to do with your use of ifevals elsewhere.